INSTALL.md

```diff
@@ -0,0 +1,159 @@
+
+
+## Grant soju user ownership of /var/lib/soju
+
+`sudo chown soju /var/lib/soju`
+
+
+## Generate certificates
+
+Using openssl generate a self signed certifcate clients will use to confirm they're connecting to your soju instance.
+
+`sudo -u soju openssl req -newkey rsa:2048 -new -nodes -x509 -days \\ 3650 -keyout /var/lib/soju/key.pem -out /var/lib/soju/cert.pem`
+
+The command will generate certificates, owned by our soju user, and write them to the soju data directory.
+
+
+## Confgiure Soju
+
+Edit the configuration file located at /etc/soju/config
+
+```
+db sqlite3 /var/lib/soju/main.db 
+log fs /var/lib/soju/logs 
+tls /var/lib/soju/cert.pem /var/lib/soju/key.pem 
+hostname ****example.org**** 
+```
+
+By default soju will listen on all interfaces. If you want it to listen only on a specific ip address add the linke `listen x.y.z.w:6697` to the config file where x.y.z.w is the ip address of that interface.
+
+
+## Create systemd service file
+
+A systemd service file will allow systemd to manage the soju daemon service. It will be started on boot, restarted on crash (giving up after a few in a row), and let you restart the service via `systemctl restart soju` Logs from the soju service can be read with `journalctl -u soju`
+
+1.  Place the below configuration in the file /etc/systemd/system/soju.service
+2.  Execute `sudo systemctl enable soju.service` which will start soju when the system boots
+3.  `sudo systemctl start soju.service` will immediately start the service.
+
+```
+[Unit] 
+Description=Soju bouncer daemon 
+After=network.target
+
+[Service] 
+Type=simple 
+Restart=always 
+RestartSec=15 
+User=soju 
+ExecStart=/usr/local/bin/soju -config /etc/soju/config
+
+[Install] 
+WantedBy=multi-user.target
+```
+
+# Using Soju
+
+## Add your first user
+
+`sudo -u soju sojuctl -config /etc/soju/config create-user ****myuser**** \\ -admin`
+
+You will be prompted for your user password. You will use this username and password to connect to your soju service via your IRC client. The -admin flag creates an admin user.
+
+## Connect to the running soju bouncer
+
+Connect with the username and password you've created in the previous step. Make sure your clienet knows to connect on port 6697 and that ssl/tsl is enabled for the connection. You will be prompted to accept the self-signed certifcate you created earlier. This will identify your soju instance to your client.
+
+
+## Adding a Server
+
+Your soju instance can be managed by messaging the bot bouncerserv. `/msg bouncerserv help` for a list of available commands.
+
+To add a new server, Libera.chat for example, use the command `/msg bouncerserv newtwork create -addr irc.libera.chat -name libera`
+
+## Set your sasl credentials
+
+`/msg bouncerserv sasl set-plain libera user pass`
+
+Notice when creating the network we named it libera, so we can use the shorter name when setting sasl commands.
+
+
+## Single Upstream vs Multi Upstream Mode
```

Comment:
Can you put/merge this and the following sections in `doc/getting-started.md` instead?